### PR TITLE
Components Style & Linters

### PR DESCRIPTION
**Pull request by VelzckC0D3**
# Game Collection (React / Redux capstone) -  `Redux` `React` `ES6` `API` `Unit Testing`

## ✨ _Requested Implementations_ ✨

✔️ _Used free to game API, from Rapid Api_
✔️ _No linter errors_
✔️ _Correct use of gitflow_
✔️ _Documented in a professional way_
✔️ _Followed design guidelines with the following logic: picked up the blue model of the APP, it haves for each page a "Hero" section that shows information related to the actual section, also the required Fonts and colors - the basic layout is followed_ 
image
image
✔️ _Credits given to the original designer on the project readme file:_
image
✔️ _Interactions for the **Home Page**: When the page loads, the web app shows the list of categories that could be filtered by words | Along with the category name, a numeric value for the actual data collected on each category is displayed | When the user clicks or taps on the category, the application navigates to the details page_
✔️ _Interactions for the **Details Page**: It retrieves data from the API to show detailed information | When a user clicks on the "Back" button, the user navigates to the Home Page_
✔️ _It's unit tested for the components, with pure and complex functions_
image
✔️ _Technical Requirements: the project is a **SPA** | The data retrieved from the API is stored in the Redux store | It filters the data using Filter stateless component | Every page haves its own route inside the SPA | Correctly deployed on Netlify [Click Here to visit Netlify Deploy](https://velzck-collection.netlify.app/)_
✔️ _And here you have the Walkthrough video recorded on Loom **(Also available on the Readme File)**_
https://www.loom.com/share/a4ab5ce8115e408fb954b92ddd5d4578

#  💡 _Thank you for your review!_ 💡

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,194 @@
...
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<!-- LIVE DEMO -->
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## 🚀 Live Demo & preview <a name="live-demo"></a>
+https://www.loom.com/share/928fd44ee8024d2ba9e7f4fd0ec1944b
```
Comment (bobb-Rob) [inline README.md]: - [ ] awesome video, I truly enjoyed your video presentation, you have good English and a proper flow of information. But there is a bit issue - > you did not show a piece of code that you are proud of as required by the project requirements. Kindly ensure you show a section of your programs, in details.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @VelzckC0D3, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (VelzckC0D3) [reply README.md]: @bobb-Rob  Thank you!! I didn't notice that part 😅 already recorded and updated!

**Review by amon-cofie — APPROVED** [review]
Hi @VelzckC0D3 , 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You have a very detailed and descriptive pull request summary
🚀 You have a professional Readme file
🚀 Great UI/UX
🚀 You used Gitflow properly
🚀 You have no linter errors


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**